### PR TITLE
Fix service finder location province sets to skip

### PR DESCRIPTION
**Pull request by megh-k**

### CODE REVIEW

File: yal/servicefinder.py [hunk 1]
```diff
@@ -559,6 +559,12 @@ async def state_save_location(self):
         return await self.go_to_state("state_category_lookup")
 
     async def state_province(self):
+        def _next(choice: Choice):
+            if choice.value == "skip":
+                return "state_validate_full_address_error"
```
Comment (KaitCrawford) [inline yal/servicefinder.py:564]: We want to allow them to skip this field. So they still need to go on to the next state and then where we store the data we should ensure we don't store "skip"

Comment (megh-k) [reply yal/servicefinder.py:564]: Ah ok, so street address and no province is fine?

Comment (KaitCrawford) [reply yal/servicefinder.py:564]: yup

**Review by KaitCrawford — APPROVED** [review]
👍

**Pull request merged**